src/incubator/f5-bigip-ctlr/README.md

```diff
@@ -21,4 +33,23 @@ Or
 helm install --set args.bigip_url=1.2.3.4 charts/src/incubator/f5-bigip-ctlr
 ```
 
+## Chart parameters:
+
+> **Note:** Helm value names cannot include the character `-` which is commonly used in the names of parameters passed to the controller. To accomodate Helm, the parameter names in `values.yaml` use `_` and then replace them with `-` when rendering.
+> e.g. `args.bigip_url` is rendered as `bigip-url` as required by the Controler.
+
+
+Parameter | Description | Default
+----------|-------------|--------
+bigip_login_secret | Secret that contains BIG-IP login credentials | f5-bigip-ctlr-login
+serviceaccount | name of ServiceAccount the ctlr should use | f5-bigip-ctlr-serviceaccount
+args.bigip_url | The admin IP for your BIG-IP | **Required**, no default
+args.partition | BIG-IP partition the ctlr is to manage | f5-bigip-ctlr
+args.log_level | Log detail | DEBUG for incubation chart
+args.verify_interval | Interval in seconds to verify BIG-IP | 2 for incubation
+args.node_poll_interval | Interval in seconds to poll the cluster | 1 for incubation
```

Comment:
Interval, in seconds, at which to poll the cluster

Reply:
Done.